packages/worker/src/worker-options.ts

```diff
@@ -93,7 +93,7 @@ export interface WorkerOptions {
    * Time to wait for pending tasks to drain after shutdown was requested.
    *
    * @format {@link https://www.npmjs.com/package/ms | ms} formatted string or number of milliseconds
-   * @default 5s
+   * @default 10s
```

Comment:
I don't see this changed in the code—was the number just wrong before?

Reply:
🤦‍♂️